conf/messages

```diff
@@ -2896,66 +2896,88 @@ inYearLosses.agent.title=Do you want to include any other Capital Gains Tax loss
 inYearLosses.trust.title=Do you want to include any other Capital Gains Tax losses made in tax year {0} to {1}?
 inYearLosses.capacitor.title=Are you including any other Capital Gains Tax losses the person made in tax year {0} to {1}?
 inYearLosses.personalRep.title=Are you including any other Capital Gains Tax losses the person made in tax year {0} to {1}?
+inYearLosses.personalRepInPeriodOfAdmin.title=Are you including any other Capital Gains Tax losses the estate made during the tax year {0} to {1}?
+inYearLosses.personalRepInPeriodOfAdmin.agent.title=Is your client including any other Capital Gains Tax losses the estate made during the tax yea {0} to {1}?
 
 inYearLosses.helpText=You make a loss if the disposal amount is less than the acquisition amount. If you include any other Capital Gains Tax losses you made in this tax year it will reduce your gain. You must have made the losses before {0}
-inYearLosses.agent.helpText=For example, your client would make a loss if they sold a property for less than they paid for it.<br/> Including any other Capital Gains Tax losses your client made in this tax year will reduce their gain. Your client must have made the losses before {0}.
+inYearLosses.agent.helpText=An example of a loss would be where your client, or the estate, sold a property or shares for less than the market value on the person’s date of death.<br/>Including any other Capital Gains Tax losses the estate made in this tax year will reduce its gain.
 inYearLosses.trust.helpText=For example, the trust would make a loss if it sold a property for less than it paid for it.<br/> Including any other Capital Gains Tax losses the trust made in this tax year will reduce its gain. The trust must have made the losses before {0}.
 inYearLosses.capacitor.helpText=For example, if a property was sold for less than it was bought for, the seller would make a loss.<br/> Including any other Capital Gains Tax losses the person made in this tax year will reduce their gain. The person must have made the losses before {0}.
 inYearLosses.personalRep.helpText=For example, if a property was sold for less than it was bought for, the seller would make a loss.<br/> Including any other Capital Gains Tax losses the person made in this tax year will reduce their gain. The person must have made the losses before {0}.
+inYearLosses.personalRepInPeriodOfAdmin.helpText=An example of a loss would be where you, or the estate, sold a property or shares for less than the market value on the person’s date of death.<br/>Including any other Capital Gains Tax losses the estate made in this tax year will reduce its gain.
+inYearLosses.personalRepInPeriodOfAdmin.agent.helpText=	An example of a loss would be where your client, or the estate, sold a property or shares for less than the market value on the person’s date of death.<br/>Including any other Capital Gains Tax losses the estate made in this tax year will reduce its gain.
```

Comment:
Does the extra white space here not get rendered on the page?

Reply:
fixed - must have hit the tab button inadvertently - it depends what component is rendering the message if that makes a difference or not.